app/assets/stylesheets/user-styles.css

```diff
@@ -28,10 +28,10 @@ img.right {
   margin: -8px; padding: 8px;
 }
 
-#sidebar .sidebar-title {
+#sidebar .sidebar-title h3{
```

Comment:
@tgaff Can you elaborate? 

#sidebar .sidebar-title h1 h2 h3{
  font-family: monospace;
}

I have applied more heading sizes to the css file, to make sure these are included. By having the headers after (.sidebar-title h1 h2 h3) does this not isolate the changes just in sidebar?